Microsoft.Toolkit.Uwp.SampleApp/SamplePages/BackgroundTaskHelper/TestBackgroundTask.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using Microsoft.Toolkit.Uwp.Notifications;
+using Windows.ApplicationModel.Background;
+using Windows.UI.Notifications;
+
+namespace Microsoft.Toolkit.Uwp.SampleApp.SamplePages
+{
+    internal class TestBackgroundTask : IBackgroundTask
+    {
+        public void Run(IBackgroundTaskInstance taskInstance)
+        {
```

Comment:
I thought it was the `Register` button in the sample app. This button does not use SPM?